### PR TITLE
[C-2478 C-2480] Add suggestedTracks

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds favorites and suggestedTracks api
Adds web-ui

image

TODO: copy coming from ux

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2465-suggested-tracks

File: packages/common/src/api/suggestedTracks.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const { data: favoritedTracks } = useGetFavoritedTrackList(
+    { currentUserId },
+    { disabled: !currentUserId }
+  )
+
+  const favoritedTracksSample = useMemo(() => {
+    return sampleSize(favoritedTracks, 5)
+  }, [favoritedTracks])
+
+  return useGetTracksByIds(
```
Comment (sliptype) [inline packages/common/src/api/suggestedTracks.ts:22]: 🔥

File: packages/web/src/pages/collection-page/components/desktop/SuggestedTracks.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <Tile className={styles.root} elevation='mid'>
+      <div className={styles.heading}>
+        <h4 className={styles.title}>{messages.title}</h4>
+        <p className={styles.description}>{messages.description}</p>
+      </div>
+      <ul>
+        {divider}
+        {!suggestedTracks && status === Status.LOADING ? (
+          <LoadingSpinner className={styles.loading} />
+        ) : null}
```
Comment (sliptype) [inline packages/web/src/pages/collection-page/components/desktop/SuggestedTracks.tsx:68]: Is there an empty state (if the user doesn't have any favorited tracks)?

**Review by sliptype — APPROVED** [review]
Good stuff 🙌

File: packages/web/src/pages/collection-page/components/desktop/SuggestedTracks.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/web/src/pages/collection-page/components/desktop/SuggestedTracks.tsx:68]: yes in that case we should pull from trending which will be done in follow up

**Pull request merged**